File: server/src/main/java/com/example/appserver/model/InstrumentBrands.java
package com.example.appserver.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Table(name = "InstrumentBrands")
public class InstrumentBrands {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    private int instrumentBrandID;
    private String instrumentBrandName;
    private String pngUrl;
}

File: android/app/src/main/java/com/example/app/model/InstrumentBrands.kt
package com.example.app.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class InstrumentBrands(
    @PrimaryKey(autoGenerate = true) val instrumentBrandID: Int,
    val instrumentBrandName: String,
    val pngUrl: String
) {
}